UKHomeOffice/sgmr-service
# Cg 216 redux store

To test

- clear localstorage and cookies/cache

- view any page
> page should load
- view localStorage
> there should be no Redux store
